cmd/commands/datasetIngestor.go

```diff
@@ -223,7 +223,7 @@ For Windows you need instead to specify -user username:password on the command l
 		}
 		color.Unset()
 		if !allowExistingSourceFolder && len(foundList) > 0 {
-			if cmd.Flags().Changed("allowexistingsource") {
+			if !cmd.Flags().Changed("allowexistingsource") {
```

Comment:
isn't the if on top enough?

Reply:
That's why I had to correct my code because I forgot to put the negation there ("if it was ***not*** set explicitly then ask the user")